index.js

```diff
@@ -1,36 +1,34 @@
 "use strict";
 
 // Return the panorama ID
-var getPanoID = function(string) {
-	let value = decodeURIComponent(string).match("^https:\/\/.*\!1s(.*)\!2e.*$");
+const getPanoID = url => {
+	let value = decodeURIComponent(url).match("^https:\/\/.*\!1s(.*)\!2e.*$");
 
 	if (value === null) {
-		throw "URL incorrect";
+		throw "Incorrect URL";
 	}
 
-	value = "F:" + value[1];
-
-	return value;
+	return "F:" + value[1];
 };
 
 // Return Embed URL to use on iframe
-var getEmbed = function(string) {
-	let id = getPanoID(string);
+const getEmbed = url => {
+	let id = getPanoID(url);
 	let embedURL = "https://www.google.com/maps/embed/v1/streetview?pano=" + id + "&key=YOUR_APIKEY";
 
 	return embedURL;
 };
 
 // Method main
-var main = function(url, options) {
+const main = (url, options) => {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/doriansampaioneto/get-streetview-panorama-id/file/2770091016/issues/source?bid=3353525&fileBranchId=3362915#l23)